README.md

```diff
@@ -1,206 +1,60 @@
-# Notary Project Overview
-
-A current overview of Notary v2, including a collection of requirements & scenarios, framing the scope of the Notary v2 project.
+# The Notary Project Overview
+
+The Notary Project aims to provide enterprise-grade solutions and cross-industry standards for securing software supply chain. The following are current repositories, aka sub-projects, under the Notary Project umbrella in an alphabetical order:
+
+| Repository                                                               | Description                                                                                                                                             |
+| ------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [.github](https://github.com/notaryproject/.github)                      | The repository that contains the Notary Project governance documents                                                                                    |
+| [meeting-notes](https://github.com/notaryproject/meeting-notes)          | The repository that contains archived meeting notes                                                                                                     |
+| [notary](https://github.com/notaryproject/notatry)                       | The repository that is an implementation of TUF that runs next to a container registry and adds the ability to sign and verify content in the registry  |
```

Comment:
This is the most clear differentiator between Notary and Notary Project, recommend leveraging this clarification through this repository when comparing and contrasting capabilities and limitations.

Reply:
Thanks @TheFoxAtWork .  

I will remove this table later, because this PR was created first before we decided to cover the Notary Project overview by the readme of `.github` repo. Could you check this PR https://github.com/notaryproject/.github/pull/32 to see if it answers your questions?

Reply:
overhauled readme.md per https://github.com/notaryproject/.github/issues/35